### PR TITLE
GitBook: [master] 29 pages and 4 assets modified

### PR DESCRIPTION
**Pull request by gravenp**
PR from Gitbook to add Introduction and Geo Web Concepts sections for docs site.

### CODE REVIEW

File: concepts/partial-common-ownership.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+1. Land holders must publicly declare a  "For Sale Price" for each of their land parcels
+2. Land holders pay a license fee to the network based on a percentage of their "For Sale Price"
+3. Any market participant can force transfer of a parcel by paying the current land holder their "For Sale Price" \(and the outstanding prepaid network fee balance\)
+
+Here's an example of how it works in practice:
+
+{% hint style="info" %}
+1\) Alice is the first Geo Web user to claim a plot of land in a park. She does so for free \(excluding blockchain transaction costs\). 
+
+2\) Alice sets her "For Sale Price" at 1 ETH. She prepays .1 ETH for one year of network fees \(10% annual rate for this example\). The expiration date of her land parcel license is set one year into the future.
```
Comment (codynhat) [inline concepts/partial-common-ownership.md:20]: I read this at first as prepaying 1.0 ETH. Might want to change to "0.1 ETH" to be more clear

**Comment by codynhat** [comment]
Not sure if this is just me, but the color of the links is very faint. It's very hard to tell when a link exists

File: geo-web-intro/why-the-geo-web.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+description: >-
+  No one. The Geo Web's protocols are closer to "laws of physics" for the
+  metaverse than a "thing" that should be privately owned or controlled.
+---
+
+# Who owns the Geo Web?
+
+No one owns the Geo Web.
```
Comment (codynhat) [inline geo-web-intro/why-the-geo-web.md:9]: What do you think about the inverse of this? Saying "everyone owns the Geo Web"? Is this more aligned with shared partial ownership and public goods?

Comment (codynhat) [reply geo-web-intro/why-the-geo-web.md:9]: Also, maybe another potential section is "What is a public good?"

**Comment by gravenp** [comment]
> Not sure if this is just me, but the color of the links is very faint. It's very hard to tell when a link exists

You're right. Changed in the theme for next push.

**Pull request merged**